src/main/attached_database.cpp

```diff
@@ -247,7 +247,7 @@ void AttachedDatabase::Close() {
 			}
 			CheckpointOptions options;
 			options.wal_action = CheckpointWALAction::DELETE_WAL;
-			storage->CreateCheckpoint(options);
+			storage->CreateCheckpoint(nullptr, options);
```

Comment:
This is the only place where we're missing out on a ClientContext, which I think is fine though.